src/compute-pr-actions.ts

```diff
@@ -156,6 +156,11 @@ interface ExtendedPrInfo extends PrInfo {
     readonly hasNewPackages: boolean;
     readonly reviewColumn: ColumnName;
     readonly isAuthor: (user: string) => boolean; // specialized version of sameUser
+
+    /**
+     * True if the author or an owner wants us to merge the PR
+     */
+    readonly mergeIsRequested: boolean;

```

Comment:
I specifically avoided separate comment lines here, to make it practical to add anything that might be needed.  (Even an in-line comment is used only in rare cases.)

Reply:
Done :heavy_check_mark: